src/styles/components/_card.scss

```diff
@@ -13,9 +13,14 @@
 .w-card {
   border-radius: 8px;
   display: block;
+  height: 462px;
```

Comment:
looks good. one small suggestion would be to use `min-height` instead here so things don't get cut off if they have a lot of text:

![image](https://user-images.githubusercontent.com/1066253/70180067-2500c600-1694-11ea-8eaa-0a4af98e94d3.png)